src/main/java/uk/gov/crowncommercial/dts/scale/cat/model/entity/ProcurementProject.java

```diff
@@ -56,7 +56,7 @@
   /**
    * Builds an instance from basic details
    *
-   * @param agreementDetails
+   * @param projectRequest
```

Comment:
should this still be `agreementDetails`?..

Reply:
Aye - probably 👍